cmake/deps.cmake

```diff
@@ -65,17 +65,17 @@ endif()
 find_package(PNG CONFIG REQUIRED)
 find_package(JPEG CONFIG REQUIRED)
 find_package(TIFF CONFIG REQUIRED)
-find_package(giflib CONFIG REQUIRED)
+#find_package(giflib CONFIG REQUIRED) # doesn't build on Android
```

Comment:
Either remove it completely or use `if(ANDROID)` condition.

Reply:
will do, but in a future pull request